amgut/templates/register_user.html

```diff
@@ -24,8 +24,12 @@
         address: "required",
         city: "required",
         state: "required",
-        zip: "required",
-        country: "required"
+        zip: {"required": true,
+              "maxlength": 10
```

Comment:
live system yelled at me for 10, so I assume the schema documentation is wrong then.